src/pages/Account/helpers.ts

```diff
@@ -11,8 +11,9 @@ type LocationUpdateData = {
   room: string | null; // legacy
 };
 
-type KidsModeUpdateData = {
-  kidsMode: boolean;
+type ToggleModeUpdateData = {
```

Comment:
When you merge you'll see the mirror video option; let's keep in line with the pattern of each checkbox being independent.